.env.example

```
@@ -61,13 +61,23 @@ AUTH_GOOGLE_SECRET=
-# APPLE_TEAM_ID=                      # 10-char Team ID
-# APPLE_KEY_ID=                       # 10-char Key ID from Sign in with Apple key
-# APPLE_PRIVATE_KEY=                  # Full .p8 contents incl. BEGIN/END lines (escape newlines as \n in single-line .env)
+# Apple Sign in — web (Auth.js) + native iOS (#286). Console: https://developer.apple.com/account/
+# Never commit the .p8 file or paste its contents into issues/PRs/commits.
+#
+# Web uses the Sign in with Apple *Services ID* as AUTH_APPLE_ID. Register return URL:
+#   https://www.still-point.me/api/auth/callback/apple
+# Apple does NOT allow localhost for web OAuth return URLs. Options: ngrok/cloudflared tunnel
+# (add tunnel URL to Services ID return URLs temporarily), or test on a Vercel preview / production.
+AUTH_APPLE_ID=
+APPLE_TEAM_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPLE_TEAM_ID key should go before the AUTH_APPLE_ID key

(UnorderedKey)